features/partners.feature

```diff
@@ -11,126 +11,193 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
+      | datum.langFormaat | 4 oktober 1986 |
+      En heeft de 'partner' de volgende 'aangaanHuwelijkPartnerschap' gegevens
+      | naam              | waarde       |
+      | datum.type        | Datum        |
+      | datum.datum       | 2021-07-20   |
+      | datum.langFormaat | 20 juli 2021 |
 
-  @gba
   Rule: Als er geen actueel huwelijk/partnerschap aanwezig is, maar wel een of meer ontbonden huwelijken/partnerschappen wordt de ex-partner met de meest recent ontbonden partnerschap geleverd.
   - Als er alleen ontbonden huwelijken/partnerschappen zijn levert de RvIG alleen de ex-partner met de meest recente datum ontbinding
   - Voor een ontbonden huwelijk of partnerschap worden de aangaanHuwelijkPartnerschap gegevens gehaald uit de meest recente historische categorie 55
 
     @gba
     Scenario: alleen ontbonden partners
-      Gegeven de persoon met burgerservicenummer 555550006 heeft de volgende huwelijken/partnerschappen in de registratie
-      | Stapel | Categorie | Burgerservicenummer (01.20) | Voornamen (02.10) | adellijke titel of predicaat (02.20) | voorvoegsel (02.30) | Geslachtsnaam (02.40) | Geboortedatum (03.10) | Geboorteplaats (03.20) | Geboorteland | Geslachtsaanduiding (04.10) | Datum aangaan (06.10) | Plaats aangaan (06.20) | Land aangaan (06.30) | Datum ontbinding (07.10) | soort verbintenis (15.10) |
-      | 1      | 5         | 555550007                   | Marieke           |                                      |                     | Groenen               | 19861004              | 0503                   | 6030         | V                           |                       |                        |                      | 20220301                 |                           |
-      | 1      | 55        | 555550007                   | Marieke           |                                      |                     | Groenen               | 19861004              | 0503                   | 6030         | V                           | 20210720              | Dunvegan               | 6039                 |                          | P                         |
-      | 2      | 5         | 555550008                   | Jan               | JH                                   | van                 | Aedel                 | 19820526              | 0599                   | 6030         | M                           |                       |                        |                      | 20191210                 |                           |
-      | 2      | 55        | 555550008                   | Jan               | JH                                   | van                 | Aedel                 | 19820526              | 0599                   | 6030         | M                           | 20171103              | 0513                   | 6030                 |                          | H                         |
+      Gegeven het systeem heeft een persoon met de volgende gegevens
```

Comment:
In regel 143 en 144 wordt hetzelfde gezegd. Een van de twee kan verwijderd worden.